### PR TITLE
Formatting amendments

### PR DESCRIPTION
**Pull request by poornimanayar**
Hi,

I have made minor formatting amendements to make the tutorial more consistent. I think improves readability as well. Let me know your thoughts :-)

Poornima

### CODE REVIEW

File: aspnetcore/data/ef-mvc/complex-data-model.md
Comment (Rick-Anderson) [inline aspnetcore/data/ef-mvc/complex-data-model.md:217]:
```suggestion
There's a one-to-zero-or-one relationship  between the `Instructor` and the `OfficeAssignment` entities. An office assignment only exists in relation to the instructor it's assigned to, and therefore its primary key is also its foreign key to the `Instructor` entity. But the Entity Framework can't automatically recognize `InstructorID` as the primary key of this entity because its name doesn't follow the `ID` or `classnameID` naming convention. Therefore, the `Key` attribute is used to identify it as the key:
```

**Review by Rick-Anderson — APPROVED** [review]

**Comment by Rick-Anderson** [comment]
Nice improvements @poornimanayar. I made one change if you want to view my [commit](https://github.com/dotnet/AspNetCore.Docs/pull/21875/commits/7a7affba1705786a656d822ac8051b955c8d27d5)
I'm doing the same thing in #21576, which is the Razor Pages version. It has alot of improvements, best practices, etc. I hope to have that published next week.

**Comment by poornimanayar** [comment]
Hi @Rick-Anderson ,

Thank you, happy to help and I quite enjoy making docs contributions! I am working through that tutorial so I am making amends as I work through them. So what is the best way to make sure your PR and mine don’t clash? And you don’t have to keep updating yours as well? 

Regards,
Poornima

**Comment by Rick-Anderson** [comment]
If you're going through [Razor Pages with Entity Framework Core](https://docs.microsoft.com/en-us/aspnet/core/data/ef-rp/intro?view=aspnetcore-5.0&tabs=visual-studio) I would hold off for another week until I merge and publish #21576

**Comment by poornimanayar** [comment]
Nope, I am going through the MVC one!

**Comment by Rick-Anderson** [comment]
In that case there are no conflicts, keep going. I highly recommend you go through the RP version when it publishes.

**Comment by poornimanayar** [comment]
I will most certainly be doing that! Already had bookmarked it in my head :-)

**Pull request merged**
